requirements.txt

```diff
@@ -2,4 +2,7 @@ feedgen>=0.9.0
 requests>=2.24.0
 beautifulsoup4==4.9.3
 sentry-sdk==1.30.0
-dateparser==1.2.0
\ No newline at end of file
+dateparser==1.2.0
+feedparser==6.0.11
+fastapi==0.111.0
+pytest==8.2.2
```

Comment:
pytest should not be installed on prod servers.
That's why it is already in a second `requirements-dev.txt` file.